olclient/openlibrary.py

```diff
@@ -196,8 +198,15 @@ def editions(self):
                 """
                 url = f'{self.OL.base_url}/works/{self.olid}/editions.json'
                 try:
-                    r = self.OL.session.get(url)
-                    editions = r.json().get('entries', [])
+                    r_json: Dict = self.OL.session.get(url).json()
+                    editions: List = r_json.get('entries', [])
+                    while True:
+                        next_page_link: Optional[str] = r_json.get('links', {}).get('next')
+                        if next_page_link is not None:
+                            r_json: Dict = self.OL.session.get(f'{self.OL.base_url}{next_page_link}').json()
```

Comment:
The same argument as above: `Dict[Any, Any]`

Suggestion: I would prefer to use string concatenation where that's the only thing we are doing: `self.OL.base_url + next_page_link`. It's easier to notice that we are only concatenating instead of interpolating.